proxy-libs/helioszerolog/helioslogger.go

```diff
@@ -0,0 +1,148 @@
+package helioszerolog
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/rs/zerolog"
+	"go.opentelemetry.io/otel/trace"
+
+	origin_zerolog "github.com/rs/zerolog"
+)
+const hsApiEndpoint = "https://app.gethelios.dev"
+
+func extractDataFromContext(otelContext context.Context, event *Event) {
+	if otelContext != nil {
```

Comment:
Can otelContext be nil? It's not a pointer

Reply:
When we are initializing the Logger we are setting the context to nil at first

Reply:
Right, but the variable type in this case isn't a pointer to a context, it's a context